sdk/core/Azure.Core.Experimental/src/DynamicData.Operators.cs

```diff
@@ -194,6 +194,30 @@ public partial class DynamicData
             }
         }
 
+        /// <summary>
+        /// Determines whether the specified <see cref="DynamicData"/> and <see cref="object"/> have the same value.
+        /// </summary>
+        /// <param name="left">The <see cref="DynamicData"/> to compare.</param>
+        /// <param name="right">The <see cref="object"/> to compare.</param>
+        /// <returns><c>true</c> if the value of <paramref name="left"/> is different to the value of <paramref name="right"/>; otherwise, <c>false</c>.</returns>
```

Comment:
Copy/paste typo?
```suggestion
        /// <returns><c>true</c> if the value of <paramref name="left"/> is equal to the value of <paramref name="right"/>; otherwise, <c>false</c>.</returns>
```

Reply:
I think you should also document the different comparisons for primitive comparisons vs. object referential comparisons.